### PR TITLE
Updating doc for contact endpoints

### PR DESCRIPTION
**Pull request by patarstar7**
Updated the contacts doc to include the new tags/contacts endpoints
- Code pr: https://github.com/GoodTimeio/EtchSchedule/pull/3902

*Updated code pr* - https://github.com/GoodTimeio/EtchSchedule/pull/3948

### CODE REVIEW

**Comment by patarstar7** [comment]
@jaspersone I changed it so that it says the correct response which is just a success

**Comment by patarstar7** [comment]
@jaspersone, I did not implement an endpoint to delete multiple tags at one time from a contact since I just went off based off the three endpoints @smarotta gave.  Should we add another endpoint for multiple tag removal given like a list of tags?

**Comment by jaspersone** [comment]
why don't we hold off then. I'd prefer to get the endpoints out sooner. thank you! do we have an ETA?

**Comment by patarstar7** [comment]
@jaspersone, hopefully sometime tomorrow? Just waiting on @smarotta and @nadapeter approvals!

File: api/v1/contacts.md
Comment (nadapeter) [inline api/v1/contacts.md]: Are these three fields actually necessary?

File: api/v1/contacts.md
Comment (nadapeter) [inline api/v1/contacts.md]: This is a bit of a weird format.
Also, we don't really want to return the relation object.
I think we should return the tags that got added.
Or simply just return "success" with no data on tag.

File: api/v1/contacts.md
Comment (nadapeter) [inline api/v1/contacts.md]: If this tag is not found what happens?
May be also document the possible error codes it may return.

File: api/v1/contacts.md
Comment (patarstar7) [reply api/v1/contacts.md]: Removed these fields!

File: api/v1/contacts.md
Comment (patarstar7) [reply api/v1/contacts.md]: Returned a list of tagIds instead

File: api/v1/contacts.md
Comment (patarstar7) [reply api/v1/contacts.md]: Also added a possible error code response for when tag is not found

**Comment by patarstar7** [comment]
I merged - https://github.com/GoodTimeio/EtchSchedule/pull/3902
@nadapeter @smarotta!

File: api/v1/contacts.md
Comment (nadapeter) [reply api/v1/contacts.md]: So if one of the tags are not found what happens?
None of the tags gets added? Only the ones that exists gets added?

File: api/v1/contacts.md
Comment (nadapeter) [inline api/v1/contacts.md]: We should probably tell them which tagIds are not found

File: api/v1/contacts.md
Comment (nadapeter) [inline api/v1/contacts.md]: The request body is in the format
```
[
    { "id": TAG_ID_1 },
    { "id": TAG_ID_2 }
]
```
but the response is in the format
```
[
    TAG_ID_1,
    TAG_ID_2
]
```

We should be consistent.

**Review by nadapeter — COMMENTED** [review]
Since this is a public API, it's really hard to change once we publish so we should try to get everything right on the first release.

@smarotta Overall thoughts on the API params and responses?

File: api/v1/contacts.md
Comment (nadapeter) [reply api/v1/contacts.md]: Also, `gtErrorCode` and `gtErrorName` were errors designed for internal API use.
I'm not sure if it's a good idea to mix the purposes.
@smarotta thoughts?

**Comment by nadapeter** [comment]
@patarstar7 Here's an example of editing multiple items in our API.
Let's try to keep the request and response structure consistent.
https://github.com/GoodTimeio/public-api-docs/blob/master/api/v1/tags.md#patch-tags

So the response for adding multiple tags should look more like
```
[
    { "id": TAG_ID_1 },
    { "id": TAG_ID_2 }
]
```

File: api/v1/contacts.md
Comment (patarstar7) [reply api/v1/contacts.md]: @nadapeter, @smarotta is off on PTO for 2 weeks I think.  I changed it so that if one of the tags are not found it gets added to an invalidTagIds list instead of returning a `gtErrorCode` and `gtErrorName`.  This way all the tags that do pass are successfully added to the contact:
```
{
    "status": "success",
    "requestId": "WrfuyI",
    "result": [
        {
            "id": "3c83269d-5f37-4607-85af-6cf0c33a284d"
        }
    ],
    "invalidTagIds": [
        {
            "id": "1b697008-502f-42bd-836c-ca0813ea4291"
        }
    ]
}
```
Please let me know if this is how we would like it formatted!

File: api/v1/contacts.md
Comment (patarstar7) [reply api/v1/contacts.md]: Ok changed it like in the above response!

**Comment by patarstar7** [comment]
@nadapeter also heres the PR associated to the code change - https://github.com/GoodTimeio/EtchSchedule/pull/3948

**Review by nadapeter — APPROVED** [review]
🚢

**Comment by nadapeter** [comment]
@patarstar7 Forgot to mention but can you update the "Table of Contents" In the top of this file to include the new endpoints?

**Comment by nadapeter** [comment]
Ignore the previous comment. I made a PR for it https://github.com/GoodTimeio/public-api-docs/pull/12

**Pull request merged**
